lesson05/assignment/css/style.scss

```diff
@@ -4,3 +4,152 @@
 @import 'settings.responsive';
 @import 'components.content';
 
+// grid
+.grid-container {
+  display: grid;
+  grid-template-areas: 
+  'header header header header header header'
+  'nav nav nav nav nav nav'
+  'side side main main main main'
+  'section section section section section section'
+  'footer footer footer footer footer footer';
+  grid-template-columns: repeat(6, 1fr);
+  @include media-query(mobile) {
+    grid-template-areas: 
+    'header header header header header header'
+    'nav nav nav nav nav nav'
+    'main main main main main main'
+    'side side side side side side'
+    'section section section section section section'
+    'footer footer footer footer footer footer';
+  }
+}
+
+// page header
+header {
+  grid-area: header;
+  color: $white;
+  background-image: linear-gradient(rgba(0, 0, 0, 0.6), rgba(0, 0, 0, 0.6)), url("https://dl2.pushbulletusercontent.com/39oo9XxNhIxmAiJYaY9ECjkESjbX7lB7/20190317_202843.jpg");
+  background-size: 100%;
```

Comment:
The background image is mostly cropped out at larger browser sizes.  You might try something like this instead:
`background-size: cover;
    background-position: center 27%;
    background-repeat: no-repeat;`